Simplified/NYPLBookAcquisitionPath.m

```diff
@@ -74,7 +78,14 @@ - (instancetype _Nonnull)initWithRelation:(NYPLOPDSAcquisitionRelation const)rel
         ContentTypeOpenAccessAudiobook,
         ContentTypeFeedbooksAudiobook,
         ContentTypeOverdriveAudiobook,
-        ContentTypeOctetStream
+        ContentTypeOctetStream,
+        ContentTypeReadiumLCP,
+        ContentTypeAudiobookZip
+      ]],
+      ContentTypeReadiumLCP: [NSSet setWithArray:@[
```

Comment:
NYPLBookAcquisitionPath verifies if the app can save a file of a certain type. When the app opens an OPDS catalog, there may be entries of different mime types. The app will show only types supported in ContentTypeOPDSCatalog  subtypes. When we receive data with ContentTypeReadiumLCP, the same way we accept only supported subtypes for that type. Anything not in these subtypes will not appear in the app (e.g., pdfs)

Reply:
but it seems like `ContentTypeReadiumLCP` and `ContentTypeAdobeAdept` can be a subtype but also a top level type... I understand why it's a subtype (because if I have a OPDS feed, I can have books of different kinds including LCP/Adobe etc) but I don't understand it as a top level... is the top level meant to describe a "feed" kind of "thing"? like an "LCP feed" where everything inside is LCP-DRM'ed?

Reply:
- When we open LCP library, we receive `ContentTypeOPDSCatalog` with `ContentTypeReadiumLCP` subtypes in it.
- When we tap an LCP-protected book in the app, the app doesn't download a book, but downloads a license file for this book `ContentTypeReadiumLCP` with content subtype of `ContentTypeEpubZip` if it is a book or `ContentTypeAudiobookZip` for an audiobook; this file is later fulfilled by LCP library and we get a real epub book or audiobook.

Reply:
so basically it seems like this data structure is for both feeds and also to describe the inner structure for a DRM-protected book, if I understand correctly.

if you can add a doc comment in the code explaining this data structure, that would be great.

and I apologize about the fact that this is not your code, I'm just trying to improve the code base readability :)

Reply:
Not exactly... this function is limiting possible content types to these subtypes. Added a comment before the subtypes

Reply:
it's still unclear what `nested and indirect acquisitions` _actually_ mean. But I think after your presentation today I can put the pieces together, let me know if I got it this time:

/////////
When we open a library that contains LCP-protected books, the app receives a feed of acquisition type `ContentTypeOPDSCatalog` that contains books of acquisition subtype `ContentTypeReadiumLCP`.
When we then try to "get" a LCP-protected book, the app first downloads a license file: the acquisition type in this case will be `ContentTypeReadiumLCP` and the subtype can only be `ContentTypeEpubZip` (for ePubs) or `ContentTypeAudiobookZip` (for audiobooks).
/////////

If this is correct can you add this to your documentation comment please?

Reply:
Yes, added this example to the documentation